ansible/roles/prosody/templates/prosody.cfg.lua.j2

```diff
@@ -108,6 +108,9 @@ modules_enabled = {
 {% if prosody_enable_end_meeting %}
       "muc_end_meeting";
 {% endif %}
+{% if prosody_enable_system_chat_message %}
```

Comment:
@quitrk 
This should be loaded under the main virtual host. 
As it says in the doc https://github.com/jitsi/jitsi-meet/blob/b90e187a7390595751202045321e13199f0d183e/resources/prosody-plugins/mod_system_chat_message.lua#L2

When loading like that the logs are full with errors:
```
Apr 19 16:48:25 modulemanager	error	Error initializing module 'system_chat_message' on 'guest.': ...prosody-plugins/mod_system_chat_message.lua:23: attempt to index a nil value (local 'token_util')
	stack traceback:
	.../prosody-plugins/mod_system_chat_message.lua:23: in main chunk
```

Can you please fix it?